.gitignore

```diff
@@ -14,3 +14,7 @@ env
 .DS_Store
 Desktop.ini
 Thumbs.db
+.coverage
+.coverage.*
+venv
```

Comment:
"Environment file" may be a better place for `venv`.

I wonder if it'd make sense to add a new "Testing files" for things like `.coverage`.

Reply:
Can we adopt this https://github.com/github/gitignore/blob/master/Python.gitignore file in pytest_watch project?